[16.0] [ADD] pos_receipt_hide_user

trisdoan opened the pull request:
Remove cashier's name on POS receipt

### Note
- The failed test is not related to this change

trisdoan committed:
[ADD] pos_receipt_hide_user

legalsylvain commented on pos_receipt_hide_user/models/pos_config.py:
```diff
@@ -0,0 +1,16 @@
+    hide_user = fields.Boolean(string="Hide User in POS receipt?", default=True)
+
+
+class ResConfigSettings(models.TransientModel):
```
nitpicking : could you split this file in two ? (one file per model).

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#L1301

ivantodorovich commented on pos_receipt_hide_user/models/pos_config.py:
```diff
@@ -0,0 +1,16 @@
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    hide_user = fields.Boolean(related="pos_config_id.hide_user", readonly=False)
```
Nitpicking / not-blocking

```suggestion
    pos_hide_user = fields.Boolean(related="pos_config_id.hide_user", readonly=False)
```

Since `res.config.settings` is a global model, used for all kinds of settings, having a prefix here would decrease the risk of overlapping with other modules

trisdoan commented:
Hello @ivantodorovich and @legalsylvain, this PR is superseded by https://github.com/OCA/pos/pull/1182
More explicitly, I renamed the module (to be more generic) and added some hiding options on the receipt.

Hence, I close this PR
